### DISCUSSION TITLE
Sofubiki – The Sofubi Wiki!

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#4077] [Showcase your Wiki]
Hey, folks! 

My little public wiki has come a long way since launch last year, and I'd love to share it with you! 

Sofubiki is a wiki dedicated to Sofubi – a vinyl toy that originated in Japan. 
We have a whole host of artists, toys and other goodies on the site and it's growing daily.

Big thanks to @NGPixel for such an awesome project! 

[sofubiki.com](https://sofubiki.com)

I'd love to hear your thoughts! 

<img src="https://sofubiki.com/logos/sofubiki-logo.png" alt="Sofubiki Logo" width="30px">

[@sofubiki](https://instagram.com/sofubiki)

**Comment by Pingu-Pox** [comment]
Nicely done!

Questions: 
1. Do you use tags? I didn't see any when trying a tag search.
2. How do you organize your pages?

**Comment by pcburcham** [comment]
Great job! Very well organized and presented 👍
